### ISSUE TITLE
# Chapter_6: example codes return different plots compared with those plots on the textbook.

### ISSUE THREAD

**Issue by FrankManc** [erykml/Python-for-Finance-Cookbook-2E#4]
Hi, I am trying to reproduce the plots from Chapter 6 on the textbook. I got everything pretty much the same as what presented on the textbook, however, there is only one plot reproduced in a different way.

That is when we train ARIMA model with additional features (exogenous variables). The example code does not return the expected plots.

<img width="988" alt="image" src="https://user-images.githubusercontent.com/104614955/227449207-2fdffeb0-57d2-4842-a3ba-74e222d72be9.png">

This is reproduced one by running the example codes. The codes works perfect and do not report any errors. What is really strange is that even if I do not put exogenous variables in the model, the model returns the exactly the same thing.

In this case, I think that exogenous variable itself might not wokring.

Thanks

**Comment by erykml** [comment]
Hi! The best way to see if the external features were incorporated into the model is to print the summary of the fitted model. If the external regressors were used, you will see them there, together with their coefficients, etc. From what I can see, they are correctly incorporated. Please let me know if you have any more doubts!

**Comment by FrankManc** [comment]
Yes, I totally agree with what you said. I checked the summary of the fitted model, I found that the external regressors were not used by the model even if I set the exogenous variables. More importantly, I used the code that you provided on the Github, which returns me the same result.

The strangest thing is that I can see that are correctly incorporated but actually they are not. Can you just try to run the Chapter 6 codes on your computer please?

**Comment by erykml** [comment]
Hey @FrankManc, apologies for the late reply. When I inspect the outcome of step 3 in the `There's more` section of the last recipe, I see the following summary output: 
<img width="524" alt="image" src="https://user-images.githubusercontent.com/30637137/233834785-bcfe893e-eda6-472c-972b-9a94d8c20e16.png">
Did you manage to solve the issue? Also, can you check explicitly before running that cell that the DataFrame actually contains the external regressors?
